geoapps_utils/numerical.py

```diff
@@ -11,12 +11,14 @@
 from scipy.spatial import Delaunay, cKDTree
 
 
-def find_curves(  # pylint: disable=too-many-locals
+def find_curves(  # pylint: disable=too-many-locals, too-many-arguments
```

Comment:
Hah!  Looks like there's even one defined on curve-apps.  Could we move that definition here?

Reply:
Yea I wanted to avoid having a utility function requiring a more complex input class structure. But I guess the argument list is getting pretty long. I can switch over.